testing/e2e/src/routes/assistant.tsx

```
@@ -0,0 +1,127 @@
+import { createFileRoute } from '@tanstack/react-router'
+import {
+  chat,
+  defineAssistant,
+  generateImage,
+  maxIterations,
+} from '@tanstack/ai'
+import { fetchServerSentEvents, useAssistant } from '@tanstack/ai-react'
+import { ChatUI } from '@/components/ChatUI'
+import { createTextAdapter } from '@/lib/providers'
+import { createImageAdapter } from '@/lib/media-providers'
+import type { Provider } from '@/lib/types'
```

ESLint

[error] `@/lib/types` type import should occur before import of `@/components/ChatUI`

(import/order)